src/tests/alerts/search_alerts.test.ts

```diff
@@ -65,239 +65,239 @@ runner.run(specFiles);
+//         alertObj.start_time = alertTimeToTimestamp.get('hDjvlhfB6jqY61T0PLK5.start_time');
+//         alertObj.last_notification_time = alertTimeToTimestamp.get(
+//           'hDjvlhfB6jqY61T0PLK5.last_notification_time',
+//         );
+//         break;
+//       default:
+//         break;
+//     }
+//   });
+
+//   fs.writeFileSync(
+//     path.join(
+//       __dirname,
+//       '..',
+//       '..',
+//       '..',
+//       'data',
+//       'indices',
+//       'alerting',
+//       '.opendistro-alerting-alerts',
+//       'documents.ndjson',
+//     ),
+//     alertObjectsList.map((line) => JSON.stringify(line)).join('\n'),
+//   );
+// }
+
+// function populateTestCaseTimes() {
+//   const templateLines = fs.readFileSync(
+//     path.join(__dirname, 'templates', 'search_alerts_test_templates.jsonl'),
+//     'utf8',
+//   );
+//   const templateLinesList = templateLines.split('\n');
+
+//   for (let i = 0; i < templateLinesList.length; i++) {
+//     templateLinesList[i] = formatString(templateLinesList[i]);
+//   }
+
+//   fs.writeFileSync(
+//     path.join(__dirname, 'specs', 'search_alerts_tests.jsonl'),
+//     templateLinesList.join('\n'),
+//   );
+// }
+
+// // only for trimming Date.toString()
+// // Date.toString() comes in format "DayOfWeek Month Day Year Time Timezone",
+// // if trimStyle "date" is passed in, this trims DayOfWeek, Time, and Timezone
+// // if trimStyle "dateAndTime" is passed in, this trims DayOfWeek, Timezone, and the seconds portion of the Time
+// function trimDate(fullDate: string, trimStyle: string) {
+//   if (trimStyle === 'date') {
+//     return fullDate.split(' ').slice(1, 4).join(' '); // trim day of week and timezone info
+//   } else if (trimStyle === 'dateAndTime') {
+//     const dateAndFullTime = fullDate.split(' ').slice(1, 5).join(' '); // trim day of week and timezone info
+//     return dateAndFullTime.substring(0, dateAndFullTime.length - 3); // trim seconds off of time
+//   }
+//   throw new Error(
+//     `passed in trimStyle was neither 'date' nor 'dateAndTime', was ${trimStyle} instead`,
+//   );
+// }
+
+// // expected syntax of a string format: <{alert-id}.{start_time|end_time|acknowledged_time|last_notification_time},{date|dateAndTime}>
+// // example: <0OgfsOs5t6yqi81f3yj18.start_time,dateAndTime>
+// function formatString(template: string) {
+//   return template.replace(/<(.*?)>/g, function (match, contents) {
+//     const contentsStr = contents as string;
+//     if (contentsStr == null) return match;
+//     const arr = contentsStr.split(',');
+//     const key = arr[0];
+//     const trimStyle = arr[1];
+//     if (alertTimeToTimestamp.has(key)) {
+//       const val = alertTimeToTimestamp.get(key);
+//       if (typeof val == 'undefined') return match;
+//       const result = trimDate(new Date(val).toString(), trimStyle);
+//       if (typeof result == 'undefined') return match;
+//       return result;
+//     }
+//     return match;
+//   });
+// }
```

Comment:
This section of code and  [`src/tests/alerts/templates`](https://github.com/opensearch-project/skills-eval/tree/main/src/tests/alerts/templates) are now unused, good to remove?

Reply:
we can remove it, probably should compare scores before and after PR to make sure

Reply:
Tested [before](https://gist.github.com/sejli/4428faeeb44f98c9082e02eb5ba0697c) and [after](https://gist.github.com/sejli/557071713d416b6345c5e4356792e153) the changes. The before is tested on `main` and the after is tested on this branch `relative-time`.